staff/pep-delbano/cv/cv.html

```diff
@@ -5,58 +5,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300i" rel="stylesheet">
     <title>Nicola's CV</title>
 </head>
 <body class="grid-container">
-    <header class="header"><h1>CURRICULUM VITAE</h1></header>
-    <img class="image" src="https://4.bp.blogspot.com/-Hw_ZYszymn8/UUuXmfD9obI/AAAAAAAABbg/acWFBmhj1d4/s1600/TESLA.jpg" alt="Nikola Tesla portrait pic">
-    <aside class="leftside">
-        <h2>Personal data</h2> 
-            <p><b>Name:</b> Nikola Tesla</p>
-            <p><b>Email:</b> nicolatesla@gmail.com</p>
-            <p><b>DOB:</b> 1989</p>
-            <p><b>Address:</b> C/ ThunderPower 12, Barcelona, Spain</p>
-    </aside><br>
 
-    <aside class="rightside1">
-        <h2>Studies</h2> 
-        <p><b>Fisics degree:</b> Superior Politecnic School at Graz </p>
-        <p><b>Electrician degree: </b>UAB</p>
+        <header class="header"><h1 class="header_maintittle">CURRICULUM VITAE</h1></header>
+
+        <img class="image" src="https://4.bp.blogspot.com/-Hw_ZYszymn8/UUuXmfD9obI/AAAAAAAABbg/acWFBmhj1d4/s1600/TESLA.jpg" alt="Nikola Tesla portrait pic">
         
-    </aside><br>
+        <aside class="leftside">
```

Comment:
aside is aimed to tangential content, but not for part of the main content, please RTFM https://www.google.es/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwjTpqO5_uTdAhVEzqQKHQclDgwQFjABegQIBBAB&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FHTML%2FElement%2Faside&usg=AOvVaw38alzYAR_ziOyIpk3tYV3z